src/sql/parse.py

```diff
@@ -217,9 +218,15 @@ def without_sql_comment(parser, line):
     return " ".join(result)
 
 
-def magic_args(magic_execute, line):
+def magic_args(magic_execute, line, cmd_from=""):
```

Comment:
seems like we're now always calling magic_args with cmd_from, right? wouldn't it be better to make it mandatory?(no default value)

Reply:
[Fixed](https://github.com/ploomber/jupysql/pull/908/files#diff-22cfc6ec3e879941494b36e0dffaf24c100ab50173c413d821ebab94d0627ddfR221)